Winapp3/Winapp3.ini

```diff
@@ -261,6 +261,44 @@ ExcludeKey66=PATH|%WinDir%\SysWOW64\config\|SOFTWARE.bak*
 ExcludeKey67=PATH|%WinDir%\SysWOW64\config\|SOFTWARE.rhk.LOG*
 ExcludeKey68=PATH|%WinDir%\SysWOW64\config\systemprofile\|NTUSER.DAT.LOG*
 ExcludeKey69=PATH|%WinDir%\SysWOW64\config\systemprofile\AppData\Local\DataSharing\Storage\|*.log
+ExcludeKey70=PATH|%WinDir%\SysWOW64\config\systemprofile\AppData\Local\|*.tmp
+ExcludeKey71=PATH|%WinDir%\SysWOW64\config\systemprofile\AppData\Local\DataSharing\Storage\|*.chk
+ExcludeKey72=PATH|%WinDir%\SysWOW64\config\systemprofile\AppData\Local\Microsoft\Windows\WebCache\|*.chk
+ExcludeKey73=PATH|%WinDir%\Panther\|*.tmp
+ExcludeKey74=PATH|%ProgramFiles%\Adobe\Adobe Media Encoder CC 2019|*.bak
+ExcludeKey75=PATH|%LocalAppData%\Packages\Microsoft.Windows.Cortana_cw5n1h2txyewy\AC\AppCache\EGFPDUEB\1\|*.txt
+ExcludeKey76=FILE|%ProgramFiles%\Adobe\Adobe Photoshop CC 2019\Required\Linguistics\Providers\Plugins2\AdobeHunspellPlugin\dictionaries\en_GB\changelog.txt
+ExcludeKey77=FILE|%ProgramFiles%\Adobe\Adobe Photoshop CC 2019\Required\Linguistics\Providers\Plugins2\AdobeHunspellPlugin\dictionaries\es_ES\Changelog.txt
+ExcludeKey78=FILE|%ProgramFiles%\Adobe\Adobe Photoshop CC 2019\Required\Linguistics\Providers\Plugins2\AdobeHunspellPlugin\dictionaries\lv_LV\Changelog.txt
+ExcludeKey79=FILE|%ProgramFiles%\Adobe\Adobe Photoshop CC 2019\Required\Linguistics\Providers\Plugins2\AdobeHunspellPlugin\dictionaries\th_th\ChangeLog.txt
+ExcludeKey80=FILE|%ProgramFiles%\Adobe\Adobe Premiere Pro CC 2019\|*.bak
+ExcludeKey81=FILE|%ProgramFiles%\ImageMagick-7.0.8-Q16\ChangeLog.txt
+ExcludeKey82=FILE|%ProgramFiles%\ImageMagick-7.0.8-Q16\www\api\MagickCore\struct__ErrorInfo.html
+ExcludeKey83=FILE|%ProgramFiles%\LibreOffice\share\config\soffice.cfg\modules\dbapp\toolbar\reportobjectbar.xml
+ExcludeKey84=FILE|%ProgramFiles%\LibreOffice\share\config\soffice.cfg\modules\dbreport\popupmenu\report.xml
+ExcludeKey85=FILE|%ProgramFiles%\LibreOffice\share\config\soffice.cfg\modules\dbreport\toolbar\reportcontrols.xml
+ExcludeKey86=FILE|%ProgramFiles%\LibreOffice\share\extensions\dict-en\changelog.txt
+ExcludeKey87=FILE|%ProgramFiles%\LibreOffice\share\extensions\dict-it\CHANGELOG.txt
+ExcludeKey88=FILE|%ProgramFiles%\Microsoft Office\Office16\SLERROR.XML
+ExcludeKey89=FILE|%ProgramFiles%\FormatFactory\updatelog.txt
+ExcludeKey90=PATH|%WinDir%\appcompat\Programs\|*.LOG;*.LOG*

```

Comment:
For "*.LOG;*.LOG*" you can just use "*.LOG*" (ExcludeKey90+91+93).

And I have to repeat: Please always use Winapp2ool > WinappDebug to make sure the syntax and sorting is correct before opening a new pull request. Thank you very much.